node/network/bridge/src/lib.rs

```diff
@@ -388,7 +386,11 @@ async fn update_our_view(
 	collation_peers: &HashMap<PeerId, PeerData>,
 ) -> SubsystemResult<()> {
 	let new_view = construct_view(live_heads.iter().map(|v| v.0), finalized_number);
-	if *local_view == new_view { return Ok(()) }
+
+	// We only want to send a view update when the heads changed, not when only the finalized block changed.
```

Comment:
I'm not sure about this, we introduced this for approval distribution: https://w3f.github.io/parachain-implementers-guide/node/approval/approval-distribution.html

>  @rphmeier: It's probably not terrible to just issue more view updates, as long as we don't issue them more 
frequently than X
Just want to make sure we don't miss anything for like 6-12 seconds at a time

the relevant issue for this is #2133

Reply:
> as long as we don't issue them more frequently than X

This was not implemented and we have spammed other nodes. 

I don't see why we need to send a view update when we finalize a block, this will also happen every 6 seconds when we are at the tip of the chain. Meaning we can just combine these updates.

Reply:
This is probably OK as long as `OurViewUpdate` doesn't get issued either and then we control the pruning of availability distribution based on that. If there are no blocks we probably want to issue an "empty" view update but we can defer that to later.